sdk/core/core-paging/package.json

```diff
@@ -35,15 +35,22 @@
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
     "build": "tsc -p .",
     "build:test": "tsc -p .",
-    "clean": "",
+    "check-format": "echo skipped",
+    "clean": "echo skipped",
     "format": "prettier --write --config .prettierrc.json \"src/**/*.ts\"",
-    "integration-test": "",
+    "integration-test:browser": "echo skipped",
+    "integration-test:node": "echo skipped",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
     "lint": "tslint -p .",
     "lint:fix": "tslint -p . --fix",
     "pack": "npm pack 2>&1",
     "prebuild": "npm run clean",
-    "test": "npm run build:test",
-    "unit-test": "npm run build:test",
+    "test:browser": "npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
+    "test:node": "npm run build:test && npm run unit-test:node && npm run integration-test:node",
+    "test": "npm run build:test && npm run unit-test && npm run integration-test",
+    "unit-test:browser": "echo skipped",
+    "unit-test:node": "echo skipped",
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser",
     "publish-preview": "mocha --no-colors && shx rm -rf dist/test && node ./.scripts/publish",
```

Comment:
This package doesn't use mocha or shx or contain this script, it must have been mistakenly copy pasted in. Let's remove it.
```suggestion
```